padl/transforms.py

```diff
@@ -510,6 +516,24 @@ def f(x):
 
         return graph
 
+    def _pd_process_traceback(self):
+        """ Find where the Transform was defined (file, lineno, file) given the traceback. """
+        a_tb = None
+        for a_tb in self._pd_traceback[::-1]:
+            if 'padl' in a_tb[0]:
```

Comment:
What is this check specifically for? And why do we need to skip parts of the traceback with `padl` in it?

Reply:
`traceback.extract_stack()` follows the lines of code for defining some object. The last element with `padl` on it contains the line where it's been defined. For example, if we create a `Transform` like
```
linear = transform(torch.nn.Linear)(6, 4)
```
The last element of the traceback with `padl` on the filename is:
```
'linear = transform(torch.nn.Linear)(6, 4)'
```